source/solvers/navier_stokes_vof_assemblers.cc

```diff
@@ -478,3 +478,57 @@ GLSNavierStokesVOFAssemblerBDF<dim>::assemble_rhs(
 
 template class GLSNavierStokesVOFAssemblerBDF<2>;
 template class GLSNavierStokesVOFAssemblerBDF<3>;
+
+
+template <int dim>
+void
+GLSNavierStokesVOFAssemblerSTF<dim>::assemble_matrix(
+  NavierStokesScratchData<dim> & /*scratch_data*/,
+  StabilizedMethodsTensorCopyData<dim> & /*copy_data*/)
+{}
+
+template <int dim>
+void
+GLSNavierStokesVOFAssemblerSTF<dim>::assemble_rhs(
+  NavierStokesScratchData<dim> &        scratch_data,
+  StabilizedMethodsTensorCopyData<dim> &copy_data)
+{
+  // Surface tension coefficient
+  const double surface_tension_coef = STF_properties.surface_tension_coef;
```

Comment:
STF = surface tension force? Please make this explicit in a comment, if it is not already done before (I may have missed that)

Reply:
I've explained it elsewhere, but added a comment in the header file too.